### ISSUE TITLE
How to convert Sigma_Profile into array

### ISSUE THREAD

**Issue by mohammad-saber** [aspuru-guzik-group/gpHSP#2]
Thank you very much for sharing your great work.

When I load data files by "pd.read_csv", the datatype of sigma_profile is String. 

Could you please guide me how to convert it into Array? 
I tried "eval()" function, but it gives the following error:
SyntaxError: invalid syntax

I think "eval()" works when there is a "," between numbers in sigma_profile. 
 
Thank you very much for your help.

**Comment by shree970** [comment]
Hi, were you able to get the gpMol package? As I saw your are using eval() functions

**Comment by mohammad-saber** [comment]
No, I don't have the package. It is not shared in the repository.

**Comment by shree970** [comment]
In paper they mention. 
"In our datasets we found that all σ-profiles can be constrained to a 61-
length continuous vector. Each vector can be interpreted as signal,
as such we normalize each vector to unit length. Since most
profiles have a similar structure, we subtract the mean signal of
all profiles to contrast the differences between profiles"
